src/helpers.ts

```diff
@@ -249,6 +273,22 @@ query getWorkspaceRepositories($workspaceName: String!, $endCursor: String) {
 }
 `;
 
+/**
+ * Checks if the linked issue is a Zenhub epic.
+ */
+export async function isIssueLinkedToEpic(
```

Comment:
Perhaps the naming should be different, as this sounds like you are checking that the given issue is linked to the epic, but in fact, you are checking that the actual given issue is the epic.